services/web/server/tests/integration/conftest.py

```diff
@@ -165,6 +166,9 @@ def app_config(_webserver_dev_config: Dict, aiohttp_unused_port) -> Dict:
     """
     cfg = deepcopy(_webserver_dev_config)
     cfg["main"]["port"] = aiohttp_unused_port()
+    cfg["db"][
```

Comment:
set ``False`` by default (in _webserver_dev_config) since this inits will be removed with #1571

Reply:
agree but I did not want to destroy something else relying on it.

Reply:
and the PR is already quite BIG